policyengine_us/variables/gov/states/wv/tax/income/subtractions/wv_other_retirement_subtraction.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class wv_other_retirement_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "West Virginia other retirement subtraction"
+    defined_for = StateCode.WV
+    unit = USD
+    definition_period = YEAR
+    reference = "https://code.wvlegislature.gov/11-21-12/"
+
+    def formula(tax_unit, period, parameters):
+        retirement_income = tax_unit("wv_retirement_income", period)
```

Comment:
Then cap at p.max_amount for each person, then sum to the tax unit.
```suggestion
        public_pension_income = person("taxable_public_pension_income", period)
```